crates/egui_kittest/src/builder.rs

```diff
@@ -0,0 +1,54 @@
+use crate::Harness;
+use egui::{Pos2, Rect, Vec2};
+
+/// Builder for [`Harness`].
+pub struct HarnessBuilder {
+    pub(crate) screen_rect: Rect,
+    pub(crate) dpi: f32,
+}
+
+impl Default for HarnessBuilder {
+    fn default() -> Self {
+        Self {
+            screen_rect: Rect::from_min_size(Pos2::ZERO, Vec2::new(800.0, 600.0)),
+            dpi: 1.0,
+        }
+    }
+}
+
+impl HarnessBuilder {
+    /// Set the size of the window.
+    #[inline]
+    pub fn with_size(mut self, size: Vec2) -> Self {
```

Comment:
slightly nice ergonomics (can pass in `[800.0, 600.0]` as an argument, for instance)

```suggestion
    pub fn with_size(mut self, size: impl Into<Vec2>) -> Self {
```